CardForm/src/test/java/com/braintreepayments/cardform/view/CardFormTest.java

```diff
@@ -123,125 +128,156 @@ public void setup_setsFlagSecure() {
     @Test
     public void setup_setsIconsForDarkBackgrounds() {
         setActivityBackground(R.color.bt_black);
-        setRequiredFields(true, true, true, true, true);
+        setRequiredFields(true, true, true, true, true, true);
 
         assertDrawableIsFromResource(R.id.bt_card_form_card_number_icon, R.drawable.bt_ic_card_dark);
+        assertDrawableIsFromResource(R.id.bt_card_form_cardholder_name_icon, R.drawable.bt_ic_cardholder_name_dark);
         assertDrawableIsFromResource(R.id.bt_card_form_postal_code_icon, R.drawable.bt_ic_postal_code_dark);
         assertDrawableIsFromResource(R.id.bt_card_form_mobile_number_icon, R.drawable.bt_ic_mobile_number_dark);
     }
 
     @Test
     public void setup_setsIconsForLightBackgrounds() {
         setActivityBackground(R.color.bt_white);
-        setRequiredFields(true, true, true, true, true);
+        setRequiredFields(true, true, true, true, true, true);
 
         assertDrawableIsFromResource(R.id.bt_card_form_card_number_icon, R.drawable.bt_ic_card);
+        assertDrawableIsFromResource(R.id.bt_card_form_cardholder_name_icon, R.drawable.bt_ic_cardholder_name);
         assertDrawableIsFromResource(R.id.bt_card_form_postal_code_icon, R.drawable.bt_ic_postal_code);
         assertDrawableIsFromResource(R.id.bt_card_form_mobile_number_icon, R.drawable.bt_ic_mobile_number);
     }
 
     @Test
     public void cardNumberIsShownIfRequired() {
-        setRequiredFields(true, false, false, false, false);
-        assertFieldsVisible(VISIBLE, GONE, GONE, GONE, GONE);
+        setRequiredFields(true, false, false, false, false, false);
+        assertFieldsVisible(VISIBLE, GONE, GONE, GONE, GONE, GONE);
     }
 
     @Test
     public void expirationIsShownIfRequired() {
-        setRequiredFields(false, true, false, false, false);
-        assertFieldsVisible(GONE, VISIBLE, GONE, GONE, GONE);
+        setRequiredFields(false, true, false, false, false, false);
+        assertFieldsVisible(GONE, VISIBLE, GONE, GONE, GONE, GONE);
     }
 
     @Test
     public void cvvIsShownIfRequired() {
-        setRequiredFields(false, false, true, false, false);
-        assertFieldsVisible(GONE, GONE, VISIBLE, GONE, GONE);
+        setRequiredFields(false, false, true, false, false, false);
+        assertFieldsVisible(GONE, GONE, VISIBLE, GONE, GONE, GONE);
+    }
+
+    @Test
+    public void cardholderNameIsShownIfRequired() {
```

Comment:
To keep the tests in order, this should probably go before the similar card number tests.